### PR TITLE
ENG-722 implement clone widget with ConfigSimpleParam

### PR DESCRIPTION
**Pull request by ichalagashvili**
This PR implement AdminConsole analogue of cloning a widget. (This PR uses ConfigSimpleParameter approach always, as seen in AdminConsole, although in the second subtask of ENG-722 native config screen will be shown for widgets that have their own unique edit screens, that's why `src/ui/widgets/clone/CloneWidgetFormContainer.js` line 49 has that empty `if` block)

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/widgets/clone/CloneWidgetFormContainer.js
Comment (jeffgo10) [inline src/ui/widgets/clone/CloneWidgetFormContainer.js:51]: is this unintended or is this like a space reserved for another subtask for ENG-722 soon?

File: src/ui/widgets/common/WidgetForm.js
Comment (jeffgo10) [inline src/ui/widgets/common/WidgetForm.js:180]: can you make a const var for 'edit'? e.g. MODE_EDIT. make sure to use this across the component file.

**Review by jeffgo10 — CHANGES_REQUESTED** [review]
I got a couple of comments

File: src/ui/widgets/clone/CloneWidgetFormContainer.js
Comment (ichalagashvili) [reply src/ui/widgets/clone/CloneWidgetFormContainer.js:51]: yes its reserved :) (please see the description of PR )

File: src/ui/widgets/common/WidgetForm.js
Comment (ichalagashvili) [reply src/ui/widgets/common/WidgetForm.js:180]: Yes, will change that!

File: src/ui/widgets/clone/CloneWidgetFormContainer.js
Comment (jeffgo10) [reply src/ui/widgets/clone/CloneWidgetFormContainer.js:51]: i knew it. consider this resolved :)

**Review by jeffgo10 — APPROVED** [review]
looks good to me

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/833#pullrequestreview-480637569" title="Approved">jeffgo10</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [jeffgo10]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
